src/models/parallelCoordinates.js

```diff
@@ -9,16 +5,18 @@ nv.models.parallelCoordinates = function() {
     // Public Variables with Default Settings
     //------------------------------------------------------------
 
-    var margin = {top: 30, right: 10, bottom: 10, left: 10}
+    var margin = {top: 30, right: 0, bottom: 10, left: 0}
         , width = null
         , height = null
         , x = d3.scale.ordinal()
         , y = {}
-        , dimensions = []
+        , dimensionsName = []
```

Comment:
Also you'll want to add a description of it in examples/documentation.html with the other option documentation (see the json at the end of the file)